readme.md

```diff
@@ -18,5 +18,13 @@ http://localhost:8080/hello?thing=Jose
 mvn test
 ```
 
+## Polyfill endpoint.
+
+http://localhost:8080/polyfill.js
+
+Parameters:
+
+- minify : If present, the polyfied features in the request will be minified. Example: http://localhost:8080/polyfill.js?minify
```

Comment:
I don't know how to do that with the servlet, wildcard is /*, however, we can achieve that by using mod_rewrite module from apache, or something similar maybe?